sdk/compute/Azure.ResourceManager.Compute/api/Azure.ResourceManager.Compute.netstandard2.0.cs

```diff
@@ -4700,43 +4787,46 @@ public partial class RestorePointInstanceView
         public Azure.ResourceManager.Compute.Models.VirtualMachineOSProfile OSProfile { get { throw null; } }
         public Azure.ResourceManager.Compute.Models.SecurityProfile SecurityProfile { get { throw null; } }
-        public Azure.ResourceManager.Compute.Models.RestorePointSourceVmStorageProfile StorageProfile { get { throw null; } }
+        public Azure.ResourceManager.Compute.Models.RestorePointSourceVmStorageProfile StorageProfile { get { throw null; } set { } }
         public string UserData { get { throw null; } }
         public string VmId { get { throw null; } }
     }
     public partial class RestorePointSourceVmDataDisk
     {
-        internal RestorePointSourceVmDataDisk() { }
+        public RestorePointSourceVmDataDisk() { }
         public Azure.ResourceManager.Compute.Models.CachingType? Caching { get { throw null; } }
-        public Azure.Core.ResourceIdentifier DiskRestorePointId { get { throw null; } }
+        public Azure.ResourceManager.Compute.Models.DiskRestorePointAttributes DiskRestorePoint { get { throw null; } set { } }
         public int? DiskSizeGB { get { throw null; } }
         public int? Lun { get { throw null; } }
-        public Azure.ResourceManager.Compute.Models.VirtualMachineManagedDisk ManagedDisk { get { throw null; } }
+        public Azure.ResourceManager.Compute.Models.VirtualMachineManagedDisk ManagedDisk { get { throw null; } set { } }
         public string Name { get { throw null; } }
+        public bool? WriteAcceleratorEnabled { get { throw null; } }
     }
     public partial class RestorePointSourceVmOSDisk
     {
-        internal RestorePointSourceVmOSDisk() { }
+        public RestorePointSourceVmOSDisk() { }
         public Azure.ResourceManager.Compute.Models.CachingType? Caching { get { throw null; } }
-        public Azure.Core.ResourceIdentifier DiskRestorePointId { get { throw null; } }
+        public Azure.ResourceManager.Compute.Models.DiskRestorePointAttributes DiskRestorePoint { get { throw null; } set { } }
         public int? DiskSizeGB { get { throw null; } }
         public Azure.ResourceManager.Compute.Models.DiskEncryptionSettings EncryptionSettings { get { throw null; } }
-        public Azure.ResourceManager.Compute.Models.VirtualMachineManagedDisk ManagedDisk { get { throw null; } }
+        public Azure.ResourceManager.Compute.Models.VirtualMachineManagedDisk ManagedDisk { get { throw null; } set { } }
         public string Name { get { throw null; } }
         public Azure.ResourceManager.Compute.Models.OperatingSystemType? OSType { get { throw null; } }
+        public bool? WriteAcceleratorEnabled { get { throw null; } }
     }
     public partial class RestorePointSourceVmStorageProfile
     {
-        internal RestorePointSourceVmStorageProfile() { }
-        public System.Collections.Generic.IReadOnlyList<Azure.ResourceManager.Compute.Models.RestorePointSourceVmDataDisk> DataDisks { get { throw null; } }
```

Comment:
System.Collections.Generic.IReadOnlyList<Azure.ResourceManager.Compute.Models.RestorePointSourceVmDataDisk> Azure.ResourceManager.Compute.Models.RestorePointSourceVmStorageProfile.DataDisks.get()' does not exist in the implementation but it does exist in the contract.	Azure.ResourceManager.Compute	

I'm not sure about how to resolve this last breaking change. Could you help @ArcturusZhang

Reply:
Here is the solution from our architect:
We need to do two things:
1. rename this property to something different, for instance, like `DataDiskList`
2. add this in a partial class to introduce the IReadOnlyList back:
```csharp
public partial class RestorePointSourceVmStorageProfile
{
[EditorBrowsable(EditorBrowsableState.Never)]
public IReadOnlyList<RestorePointSourceVmDataDisk> DataDisks => DataDiskList;
}
```
Reason:
Since the ApiCompat tool is not allowing to change the type of this, we have to keep the original property and hide it (by adding EBNever attribute). To do this, we have to change the name of this property so that the generator will not let the property with the new type taking the old name.
Returning the new property since their types are actually compatible and we never want to let these two properties out of sync or take extra memory

Reply:
so i am trying to rename the property, and with show serialized name, it shows: 
Serialized Name: RestorePointSourceVMStorageProfile.dataDisks

and when I put in autorest.md in rename: 
RestorePointSourceVmStorageProfile.dataDisks: DataDiskList 
its not working for some reason

Reply:
do you know why that might be ? @ArcturusZhang

Reply:
I will take a look on why this renaming is not working - this might be a bug that we need to fix.

Reply:
tried updating the name using directives and it also did not work. 
I tried hardcoding the rename, and adding the above customization, and I get this error: 
![image](https://user-images.githubusercontent.com/10857691/235003684-54d69ee4-76f0-4579-bcc0-b9232a039f5a.png)

Reply:
Requesting more urgent support to get a fix on this please ! its been hindering our new version release for some time now

Reply:
resolved by Dapeng. Thank you